### ISSUE TITLE
weaken 'complex regex' cop

### ISSUE THREAD

**Issue by dirkbolte** [rubocop/rubocop#391]
The following code results in a `Use %r for regular expressions matching more than 1 '/' character.` warning:

``` ruby
# encoding: UTF-8
puts /\//m =~ '/'
```

It works fine when removing the `m`.

I was wondering whether the cop is intended to already report this as complex regex?

rubocop version: 0.10.0
Parser version: 2.0.0.pre2
ast version: 1.1.0
